maven-surefire-common/src/test/java/org/apache/maven/plugin/surefire/AbstractSurefireMojoTest.java

```diff
@@ -2183,6 +2183,13 @@ public void setToolchain( Toolchain toolchain ) throws Exception
             toolchainField.setAccessible( true );
             toolchainField.set( this, toolchain );
         }
+
+        public void setJvm( String jvm ) throws Exception
+        {
+            Field toolchainField = AbstractSurefireMojo.class.getDeclaredField( "jvm" );
```

Comment:
pls substitute these three lines with `setInternalState( this, "jvm", jvm );`

Reply:
It becomes static import `import static org.powermock.reflect.Whitebox.setInternalState;`

Reply:
That's much better.